Classes/Domain/Model/Forum/Topic.php

```diff
@@ -24,14 +24,14 @@
  *  This copyright notice MUST APPEAR in all copies of the script!      *
  *                                                                      */
 
-use Mittwald\Typo3Forum\Configuration\ConfigurationBuilder;
```

Comment:
Are there any remaining usages of the `Mittwald\Typo3Forum\Configuration\ConfigurationBuilder` class? If not, the class should be removed entirely.